Spring-Boot/Spring-Boot-Problem/src/test/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemControllerTest.java

```diff
@@ -367,11 +367,28 @@ void Given_1_ProblemsSaved_WhenGetProblemsByIds_1_2_ShouldRespondThat_1() throws
         problemsShouldHaveIds(actualProblems, 1);
     }
 
+    @Test
+    void GivenOneProblemCreated_WhenDeleteById_ShouldDeleteSuccessfully() throws Exception {
```

Comment:
`GivenOneProblemCreated_WhenDeleteIt_ShouldArchiveItSuccessfully`

Reply:
sorry, I mistype this one, should be `GivenOneProblemCreated_WhenArchiveIt_ShouldArchiveItSuccessfully`